### ISSUE TITLE
Requiring in renderer process isn't working with CRA

### ISSUE THREAD

**Issue by joshuawootonn** [megahertz/electron-log#186]
Hello!

## TLDR: 
I am Using create react app  in electron and when I `require('electron-log')` in one of my react components I get errors based on things in process not being undefined. 

Like this: 
![image](https://user-images.githubusercontent.com/20950876/76231342-1fff6580-61f3-11ea-8550-f647aad50f8f.png)

I can manually set things right before I require, but its a rabbit trail.
![image](https://user-images.githubusercontent.com/20950876/76231440-402f2480-61f3-11ea-958c-a32fd4c5a271.png)

I think that this comes down to the fact that create react app is made with the intent of it being agnostic of it's environment, but I was wondering if you have a work around.

## Context
Followed[ this guide in for electron setup ](https://www.codementor.io/@randyfindley/how-to-build-an-electron-app-using-create-react-app-and-electron-builder-ss1k0sfer)

## What I have tried
1. Both options under [your docs]( https://github.com/megahertz/electron-log/blob/master/docs/node-integration.md): 
2. Redefining process in above the require... rabbit hole

FWIW this also happens when I build.
![image](https://user-images.githubusercontent.com/20950876/76232189-5ab5cd80-61f4-11ea-803d-6593b00ee7b9.png)

**Comment by megahertz** [comment]
Thank you for the report. Will fix in the next release.

**Comment by joshuawootonn** [comment]
Fixed by allowing node integration and using `window.require` in the rendering process instead of just `require`
![image](https://user-images.githubusercontent.com/20950876/76260492-e7c54a80-6225-11ea-85d5-2f6b0d582324.png)
